civis/civis.py

```diff
@@ -379,10 +381,16 @@ class APIClient(MetaMixin):
         a local cache of the specification may be passed as either an
         OrderedDict or a filename which points to a json file.
     """
-    @deprecate_param('v2.0.0', 'resources')
+    @deprecate_param('v2.0.0', 'retry_total', 'resources')
     def __init__(self, api_key=None, return_type='snake',
                  retry_total=6, api_version="1.0", resources="all",
                  local_api_spec=None):
+        if retry_total != 6:
```

Comment:
If someone explicitly passes in the kwarg+value of `retry_total=6`, then this wouldn't trigger the warning -- I'm okay with not handling this, since few people if anyone would actually set this kwarg, and that if they did at all it'd be a different value than 6, which would trigger the warning.